content/post/agile-analytics-p3.md

```diff
@@ -0,0 +1,91 @@
+---
+author: Michael Kaminsky
+title: "Agile Analytics, Part 3: The Adjustments"
+publishDate: "2018-07-01"
+draft: true
+tags: 
+ - agile
+ - scrum
+ - process
+---
+
+
+Agile software engineering practices have become the standard work management tool for modern software development teams. Are these techniques applicable to analytics, or is the nature of research prohibitively distinct from the nature of engineering?
+
+In this post I discuss some adjustments to the scrum methodology to make the process work better for analytics and data science teams.
+
+<!--more-->
+
+If you haven't already read them, you can find
+
+* [part 1 here]({{< relref "agile-analytics-p1.md" >}})
+* [part 2 here]({{< relref "agile-analytics-p1.md" >}})
```

Comment:
`agile-analytics-p2.md` (unless it's here as a placeholder so the link works..?)